agent/src/Handler/V0.hs

```diff
@@ -93,6 +93,7 @@ getSpecs settings = do
     specsDisk       <- fmap show . metricDiskSize <$> getDfMetrics
     specsNetworkId  <- runM $ injectFilesystemBaseFromContext settings getStart9AgentHostname
     specsTorAddress <- runM $ injectFilesystemBaseFromContext settings getAgentHiddenServiceUrl
+    specsLanAddress <- fmap (<> ".local") <$> runM $ injectFilesystemBaseFromContext settings getStart9AgentHostname
```

Comment:
```suggestion
    specsLanAddress <- fmap (<> ".local") . runM $ injectFilesystemBaseFromContext settings getStart9AgentHostname
```

Reply:
This is going to take someone else an unreasonable amount of time to figure out that this is fmap for the function functor.

Reply:
I didn't even know, the type error without it was THERE'S ANTHER FUNCTOR HERE BRO so I added the fmap